### PR TITLE
MOB-1637 Inner spacing url bar

### PR DESCRIPTION
**Pull request by d4r1091**
[MOB-1637](https://ecosia.atlassian.net/browse/MOB-1637)

## Context

There is a misalignment in spacing between the right and left-hand sides of the URL Bar.
Both should be equally aligned.

## Approach

After a quick review of how the `TabLocationView` is made, I've come to the realization that a refactor would take too much time in order to simplify the architecture of the URL Bar as well as bring the fix in.
So I did come up with a compromise, leveraging the minimum impact as well as achieving the expected behavior.

## Other

- This UX alignment also fixes an issue when the reading mode gets triggered. Because of the URL being `localhost../reader-mode` the business logic treated it as not secure, hence showing the ⚠️ icon.
- Also moved our previous computer var URL extension into the Ecosia's domain for simplifying upgrades.
 
Simulator Screen Recording - iPhone 14 - 2023-06-07 at 14 33 01

### CODE REVIEW

File: Client/Frontend/Toolbar+URLBar/TabLocationView.swift [hunk 1]
```diff
@@ -66,8 +66,10 @@ class TabLocationView: UIView {
     }
 
     lazy var urlTextField: URLTextField = .build { urlTextField in
+        /* Ecosia: removing obsolete implementation as we don't support 4S anymore
         // Prevent the field from compressing the toolbar buttons on the 4S in landscape.
```
Comment (lucaschifino) [inline Client/Frontend/Toolbar+URLBar/TabLocationView.swift:70]: Very minor and no blocker: 
I confess I'm a bit confused with the comment within a comment. I understand the idea is that the outer comment "negates" the inner one, but in that case is there a reason to keep the inner one at all?

**Review by lucaschifino — DISMISSED** [review]
Looks good to me! 🚀 

Added only a minor non-blocking comment 👍

File: Client/Frontend/Toolbar+URLBar/TabLocationView.swift (hunk 1 above)
Comment (d4r1091) [reply Client/Frontend/Toolbar+URLBar/TabLocationView.swift:70]: Total refuse! Thanks for checking!

**Review by lucaschifino — APPROVED** [review]
LGTM 🟢

**Pull request merged**